ChangeLog.d/max-record-payload-api.txt

```diff
@@ -0,0 +1,6 @@
+API changes
+   * Remove `mbedtls_ssl_get_{input,output}_max_frag_len()`
```

Comment:
The ChangeLog is still plaintext, not markdown (though I agree we may want to change in the future). Also, perhaps we should avoid abbreviations such as `{intput,output}` and just spell things out fully (at least the first time).

Reply:
Ok. It looks very clumsy now, but it doesn't really matter anyway.